src/zcl/definition/cluster.ts

```diff
@@ -1900,6 +1900,17 @@ const Cluster: {
             viessmannWindowOpenForce: {ID: 0x4003, type: DataType.boolean, manufacturerCode: ManufacturerCode.VIESSMAN_ELEKTRO},
             viessmannAssemblyMode: {ID: 0x4012, type: DataType.boolean, manufacturerCode: ManufacturerCode.VIESSMAN_ELEKTRO},
             schneiderWiserSpecific: {ID: 0xe110, type: DataType.enum8, manufacturerCode: ManufacturerCode.SCHNEIDER},
+            danfossWindowOpenInternal: {ID: 0x4000, type: DataType.enum8, manufacturerCode: ManufacturerCode.DANFOSS},
```

Comment:
@Koenkk should these be sorted on attribute id?

Reply:
Yes, but grouping all the danfoss attributes also makes sense so this is fine.